ccmlib/scylla_node.py

```diff
@@ -38,6 +39,52 @@
 if TYPE_CHECKING:
     from ccmlib.scylla_cluster import ScyllaCluster
 
+class ScyllaKeyType:
+    """
+    Helper class to define Scylla Key definition with Type strings.
+    Use the cassandra class name from https://github.com/scylladb/scylladb/blob/master/docs/dev/cql3-type-mapping.md as type strings.
+    Examples :
+        type1 = ScyllaKeyType.make_partition_key("Int32Type")
+        type2 = ScyllaKeyType.make_clustering_key("Int32Type", "FloatType")
+    """
+    class TypeKind(Enum):
+        REGULAR_KEY = 1
```

Comment:
Just `REGULAR`, see above.
Also apply the change to other places where `regular` is used.

Reply:
Done.